src/App.vue

```diff
@@ -0,0 +1,40 @@
+<template lang="pug">
+  #app
+    Calculator
+</template>
+
+<script>
+import Calculator from "./components/Calculator.vue";
+
+export default {
+  name: "App",
+  components: {
+    Calculator
+  },
+  data() {
+    return {};
+  },
+  created() {
+    fetch(`https://source.unsplash.com/1600x900/?beach`).then(response => {
+      this.backgroundImage = response.url;
+      document.body.style.backgroundColor = "white";
+      document.body.style.backgroundImage = "url('" + response.url + "')";
+    });
+  }
```

Comment:
Could have used async/await instead of `.then`
Also, any promises should have error handling with `.catch`